backend/Dockerfile

```diff
@@ -27,4 +24,4 @@ EXPOSE 3000
 
 # Run the application. -L for legacy mode to enable hot changes in container
 # https://github.com/remy/nodemon?tab=readme-ov-file#application-isnt-restarting
-CMD ["nodemon", "-L"]
+CMD ["npm", "run", "dev"]
```

Comment:
We'll probably need to make a similar change in the frontend dockerfile and compose.yaml later. It's not super urgent because we're not converting it to TS right now.